src/rebar_prv_install_deps.erl

```diff
@@ -224,28 +224,13 @@ update_pkg_deps(Profile, Pkgs, Packages, Upgrade, Seen, State, Locks) ->
 handle_pkg_dep(Profile, Pkg, Packages, Upgrade, DepsDir, Fetched, Seen, Locks, State) ->
     IsLock = pkg_locked(Pkg, Locks),
     AppInfo = package_to_app(DepsDir, Packages, Pkg, IsLock, State),
+    Deps = rebar_app_info:deps(AppInfo),
     Level = rebar_app_info:dep_level(AppInfo),
     {NewSeen, NewState} = maybe_lock(Profile, AppInfo, Seen, State, Level),
     {_, AppInfo1} = maybe_fetch(AppInfo, Profile, Upgrade, Seen, NewState),
-
-    Profiles = rebar_state:current_profiles(State),
-    Name = rebar_app_info:name(AppInfo1),
-    C = rebar_config:consult(rebar_app_info:dir(AppInfo1)),
-    BaseDir = rebar_state:get(State, base_dir, []),
-    S1 = rebar_state:new(rebar_state:set(rebar_state:new(), base_dir, BaseDir),
-                        C, rebar_app_info:dir(AppInfo1)),
-    S2 = rebar_state:apply_overrides(S1, Name),
-
-    Plugins = rebar_state:get(S2, plugins, []),
-    S3 = rebar_state:set(S2, {plugins, Profile}, Plugins),
-
-    S4 = rebar_state:apply_profiles(S3, Profiles),
-    AppInfo2 = rebar_app_info:state(AppInfo1, S4),
-
-    %% Dep may have plugins to install. Find and install here.
-    S5 = rebar_plugins:install(S4),
-    AppInfo3 = rebar_app_info:state(AppInfo2, S5),
-
```

Comment:
Right, it is done in `handle_dep`. There is more I plan to do to bring the 2 using similar code paths so there is less duplication that could cause bugs.